source/common/quic/envoy_quic_server_connection.cc

```diff
@@ -17,7 +17,13 @@ EnvoyQuicServerConnection::EnvoyQuicServerConnection(
     : quic::QuicConnection(server_connection_id, initial_self_address, initial_peer_address,
                            &helper, &alarm_factory, writer, owns_writer,
                            quic::Perspective::IS_SERVER, supported_versions),
-      QuicNetworkConnection(std::move(connection_socket)) {}
+      QuicNetworkConnection(std::move(connection_socket)) {
+  if (Runtime::runtimeFeatureEnabled(
+          "envoy.reloadable_features.quic_defer_send_in_response_to_packet")) {
+    set_defer_send_in_response_to_packets(GetQuicFlag(FLAGS_quic_defer_send_in_response));
+    defer_send_ = true;
```

Comment:
out of curiosity, instead of adding a new envoy-specific member, would it make sense to expose `defer_send_in_response_to_packets_` via a getter on QuicConnection?

Reply:
That's also possible.

Reply:
No need to block this PR, but care to cook up a quiche CL and eventually switch over to using it?

Reply:
Oh, you already landed this CL! Nice!